pkgs/development/interpreters/icon-lang/default.nix

```diff
@@ -12,6 +12,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = stdenv.lib.optionals withGraphics [ libX11 libXt ];
 
+  patches = [
+    # Adapted from https://github.com/gtownsend/icon/commit/bfc4a6004d0d3984c8066289b8d8e563640c4ddd
```

Comment:
Did you try this?
```
patches = [
  (fetchpatch {
    url = "https://github.com/gtownsend/icon/commit/bfc4a6004d0d3984c8066289b8d8e563640c4ddd";
    sha256 = ...
    excludes = [
      "doc/relnotes.htm"
      "src/h/version.h"
    ];
  })
];
```

Hopefully the `#  Contributor:  Cheyenne Wills` isn't going to be an issue :crossed_fingers: